client/components/LayoutEditor/LayoutEditorCollectionHeader/getMetaDataChecklist.tsx

```diff
@@ -0,0 +1,205 @@
+				checked={printPublicationDate ? !hideIssuePrintPublicationDate : false}
+				onChange={() =>
+					onChange({ hideIssuePrintPublicationDate: !hideIssuePrintPublicationDate })
+				}
+				label="Print Publication Date"
+			/>
+			<Checkbox
+				disabled={!publicationDate}
+				checked={publicationDate ? !hideIssuePublicationDate : false}
+				onChange={() => onChange({ hideIssuePublicationDate: !hideIssuePublicationDate })}
+				label="Publication Date"
+			/>
+		</>
+	);
+};
+
+const PreviewBookElementFields = (props: Props) => {
+	const { content, onChange, collection } = props;
+	const { hideDoi, hideIsbn, hideCopyrightYear, hideBookPublicationDate, hideEdition } = content;
+	const { isbn, copyright, published, edition } = BookMetadata(collection);
+	const doi = getCollectionDoi(collection);
+
+	return (
+		<>
+			<Checkbox
+				disabled={!doi}
+				checked={doi ? !hideDoi : false}
+				onChange={() => onChange({ hideDoi: !hideDoi })}
+				label="DOI"
+			/>
+			<Checkbox
+				disabled={!isbn}
+				checked={isbn ? !hideIsbn : false}
+				onChange={() => onChange({ hideIsbn: !hideIsbn })}
+				label="ISBN"
+			/>
+			<Checkbox
+				disabled={!copyright}
+				checked={copyright ? !hideCopyrightYear : false}
+				onChange={() => onChange({ hideCopyrightYear: !hideCopyrightYear })}
+				label="Copyright Year"
+			/>
+			<Checkbox
+				disabled={!published}
+				checked={published ? !hideBookPublicationDate : false}
+				onChange={() => onChange({ hideBookPublicationDate: !hideBookPublicationDate })}
+				label="Publication Date"
+			/>
+			<Checkbox
+				disabled={!edition}
+				checked={edition ? !hideEdition : false}
+				onChange={() => onChange({ hideEdition: !hideEdition })}
+				label="Edition"
+			/>
+		</>
+	);
+};
+
+const PreviewConferenceElementFields = (props: Props) => {
+	const { content, onChange, collection } = props;
+	const { hideDoi, hideTheme, hideAcronym, hideConferenceDate, hideLocation } = content;
+	const { theme, acronym, location, conferenceDate } = ConferenceMetadata(collection);
+	const doi = getCollectionDoi(collection);
+
+	return (
+		<>
+			<Checkbox
+				disabled={!doi}
+				checked={doi ? !hideDoi : false}
+				onChange={() => onChange({ hideDoi: !hideDoi })}
+				label="DOI"
+			/>
+			<Checkbox
+				disabled={!theme}
+				checked={theme ? !hideTheme : false}
+				onChange={() => onChange({ hideTheme: !hideTheme })}
+				label="Theme"
+			/>
+			<Checkbox
+				disabled={!acronym}
+				checked={acronym ? !hideAcronym : false}
+				onChange={() => onChange({ hideAcronym: !hideAcronym })}
+				label="Acronym"
+			/>
+			<Checkbox
+				disabled={!location}
+				checked={location ? !hideLocation : false}
+				onChange={() => onChange({ hideLocation: !hideLocation })}
+				label="Location"
+			/>
+			<Checkbox
+				disabled={!conferenceDate}
+				checked={conferenceDate ? !hideConferenceDate : false}
+				onChange={() => onChange({ hideConferenceDate: !hideConferenceDate })}
+				label="Conference Date"
+			/>
```

Comment:
Some repetition in these components that we might be able to handle programmatically, lets pair program something during our chat later?